src/sentry/static/sentry/app/components/events/eventEntries.tsx

```diff
@@ -55,16 +57,41 @@ export const INTERFACES = {
   spans: SpansInterface,
 };
 
-class EventEntries extends React.Component {
+const defaultProps = {
+  isShare: false,
+  showExampleCommit: false,
+  showTagSummary: true,
+};
+
+// Custom shape because shared view doesn't get id.
+type EventEntriesOrganization = {
+  id?: string;
+  slug: string;
+  features?: Array<string>;
+};
+
+type Props = {
+  // This is definitely required because this component would crash if
+  // organization were undefined.
+  organization: EventEntriesOrganization;
```

Comment:
According to the `OrganizationSummary` the `id` and `features` props are not optional...are you sure about making these props optional?

Reply:
This is how the original JS code behaves and the code comment is about the shared issues view which is special. I am not sure what the cleanest way to reconcile is but the stuff is definetly missing in the API sometimes

Reply:
Should this be named SharedViewOrganization to make that restriction clearer?

Reply:
done, thx mark